Source/WebKit/Platform/IPC/Attachment.h

```diff
@@ -83,6 +93,9 @@ class Attachment {
     mach_msg_type_name_t disposition() const { return m_disposition; }
 #elif OS(WINDOWS)
     HANDLE handle() const { return m_handle; }
+#elif PLATFORM(HAIKU)
+	team_id connectionID () const { return m_connectionID; }
```

Comment:
encoder for long int is not provided maybe we should add it to Encoder? or we can manually encode required stuff in attachments

Reply:
Casting the team_id to unsigned int or uint32_t should work. But you should be able to use a ProcessID as well here?